ts/initializeTodos.ts

```diff
@@ -0,0 +1,57 @@
+import { todos } from "./storage-todos.js";
+import { storage } from "./storage.js";
+import { Todo } from "./types.js";
+
+const activeTab = document.querySelector("#active-button") as HTMLElement;
+const completedTab = document.querySelector("#completed-button") as HTMLElement;
+
+function initializeTodos() {
+  const allCheckBoxes = getCheckBoxes();
+  allCheckBoxes.forEach((checkBox) => {
+    const currentTodo = checkBox.closest(".todo-ul-item") as HTMLElement;
+    const currentTodoId = currentTodo.id;
+    const label = currentTodo.firstElementChild as HTMLElement;
+    todos.forEach((todo) => {
+      if (todo.id !== currentTodoId) return;
+      if (todo.completed) {
+        checkBox.checked = true;
+        label.style.textDecoration = "line-through";
+      } else {
+        checkBox.checked = false;
+      }
+    });
+
+    checkBox.addEventListener("click", () => {
+      const isActiveTab = activeTab.classList.contains("active");
+      const isCompletedTab = completedTab.classList.contains("active");
+      const shouldDisplay =
```

Comment:
You decided to display todos by changing their style which is ok approach. You are doing it manually for every todo is this logic seems quite complex. Have you considered about css ? Basically, the requirement that we need: List of todos depends on current tab.

If active, show only active todos.
If completed, show only completed todos with trash cans. 
If all, show all todos but without trash cans.

we can do it via css I think. While switching tab, just assign needed class for your list. Then you can control todos style inside the list via css (make them display: 'block' or 'none').

For example:

```
.todos-list.active .todo-completed {
  display: none; // hiding completed todos when list is active
}
```